arkouda/pdarrayclass.py

```diff
@@ -1974,7 +1975,7 @@ def argmax(pda: pdarray) -> np.int64:
     RuntimeError
         Raised if there's a server-side error thrown
     """
-    repMsg = generic_msg(cmd="reduction", args="{} {}".format("argmax", pda.name))
+    repMsg = generic_msg(cmd="reduction", args={"op": "argmax", "array": pda.name})
```

Comment:
Is the `.name` required here due to `Datetime` issues as well?

Reply:
Removed `name` since the ticket adding Datetime parsing got merged